Gemfile

```diff
@@ -4,7 +4,6 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'hana', '~> 1.3'
```

Comment:
Was this meant to be dropped?

Reply:
It is duplicated in gemspec which for a librsry is the correct location .

Reply:
It is duplicated in gemspec which for a librsry is the correct location .